management/signing_key_test.go

```diff
@@ -0,0 +1,72 @@
+package management
+
+import (
+	"testing"
+)
+
+func TestKey(t *testing.T) {
```

Comment:
I might be nitpicking here @paddycarey but it feels "TestSigningKey" feels more specific versus "TestKey". 

Thoughts @paddycarey?

Reply:
makes sense, updated!